### PR TITLE
Ruby 2.4.6 Released (pt)

### PR DESCRIPTION
**Pull request by jcserracampos**

### CODE REVIEW

File: pt/news/_posts/2019-04-01-ruby-2-4-6-released.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+e começará a fase de manutenção de segurança dele.
+Isto signifa que após o lançamento da 2.4.6 nós nunca traremos qualquer correção de bug
+para a 2.4, exceto correções de segurança.
+O período da fase de manutenção de segurança está programda para 1 ano.
+Ao fim deste período, o suporte oficial ao Ruby 2.4 terminará.
+Assim sendo, nós recomendamos que você comece a planejar para migrar para o Ruby 2.6 ou 2.5
+
+## Download
+
+* [https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2](https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.6.tar.bz2)
```
Comment (JuanitoFatas) [inline pt/news/_posts/2019-04-01-ruby-2-4-6-released.md]: Please follow https://github.com/ruby/www.ruby-lang.org/pull/2026 to fix the download section, thanks :-)

Comment (jcserracampos) [reply pt/news/_posts/2019-04-01-ruby-2-4-6-released.md]: I will amend asap.

Comment (jcserracampos) [reply pt/news/_posts/2019-04-01-ruby-2-4-6-released.md]: @JuanitoFatas, done

**Comment by JuanitoFatas** [comment]
Thanks Júlio, merged as df0d422 :-)

**Pull request closed without merging**